cocli/data/corim/templates/corim-cca-realm.json

```diff
@@ -0,0 +1,25 @@
+{
+    "corim-id": "5c57e8f4-46cd-421b-91c9-08cf93e13cfc",
+    "dependent-rims": [
+        {
+            "href": "https://parent.example/rims/ccb3aa85-61b4-40f1-848e-02ad6e8a254b",
+            "thumbprint": "sha-256:5Fty9cDAtXLbTY06t+l/No/3TmI0eoJN7LZ6hOUiTXU="
+        }
+    ],
```

Comment:
This CoRIM feature whose functioning is not especially clear to me ;-) I wouldn't want to trick people into thinking that it's a necessary feature for CCA endorsements.